admin/src/components/ContributionMessages/ContributionMessagesFormular.vue

```diff
@@ -38,10 +38,12 @@ export default {
       form: {
         text: '',
       },
+      isSubmitting: false,
```

Comment:
do we not use `loading` normally? Please check with @Mogge if this is correct naming.

Reply:
You are right, we normally use `loading`.
This is fixed by https://github.com/gradido/gradido/commit/403e68480c0cb7efa3f84544028398fb5a774d76.